src/services/fetchers/PolygonIOCurrencySnapshotGramsFetcher.ts

```diff
@@ -21,15 +25,36 @@ class PolygonIOCurrencySnapshotGramsFetcher extends BasePolygonIOSingleFetcher {
     this.valuePath = '$.ticker.lastQuote.a';
   }
 
-  async apply(params: {ticker: string}): Promise<number> {
+  async apply(params: {ticker: string}, options: FeedFetcherOptions): Promise<number> {
```

Comment:
looks like this was missed because we not have intefrace here, please apply interface for this fetcher